index.html

```diff
@@ -169,31 +209,52 @@
                         <p>Project Outside-IN partners will include:</p>
                         
                         <ul>
-                        
                             <li> <a href="http://www.yes.on.ca/" title="Youth Employment Services" target="_blank"> Youth Employment Services </a> </li>
                             <li> <a href="https://www.tigweb.org/" title="TakingIT Global" target="_blank"> TakingIT Global </a> </li>
```

Comment:
Based on their website, there should not be a space between TakingIT and Global. Same goes for the longer partner description below.